app/scripts/templates/sign_in.mustache

```diff
@@ -19,9 +19,13 @@
     {{^error}}
       <div class="error"></div>
       <div class="success"></div>
-        {{#isMigration}}
+      {{#isSyncMigration}}
           <div class="info nudge">{{#t}}Migrate your sync data by signing in to your Firefox&nbsp;Account.{{/t}}</div>
-        {{/isMigration}}
+      {{/isSyncMigration}}
+
+      {{#isAmoMigration}}
+          <div class="info nudge">{{#t}}Migrate your Mozilla Add-ons data by signing in to your Firefox&nbsp;Account.{{/t}}</div>
```

Comment:
> but don't show anything for other migrations (which is what I think @shane-tomlinson is suggesting in #3399 (comment))

Yes, @ryanfeeley doesn't show anything on the signin page for the AMO migration. The `isAmoMigration` section can be removed.